config/crd/bases/osp-director.openstack.org_openstackprovisionservers.yaml

```diff
@@ -6,20 +6,21 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.3.0
   creationTimestamp: null
-  name: provisionservers.osp-director.openstack.org
+  name: openstackprovisionservers.osp-director.openstack.org
 spec:
   group: osp-director.openstack.org
   names:
-    kind: ProvisionServer
-    listKind: ProvisionServerList
-    plural: provisionservers
-    singular: provisionserver
+    kind: OpenStackProvisionServer
+    listKind: OpenStackProvisionServerList
+    plural: openstackprovisionservers
+    singular: openstackprovisionserver
   scope: Namespaced
   subresources:
     status: {}
   validation:
     openAPIV3Schema:
-      description: ProvisionServer is the Schema for the provisionservers API
+      description: OpenStackProvisionServer is the Schema for the provisionservers
```

Comment:
Should `provisionserver` here and below be renamed?

Reply:
I wasn't going to change these parameter names as part of this patch.

Reply:
Perhaps we can debate the pros/cons of this?

Reply:
oh, I see. This is just a comment. So yeah, lets fix it!